Tests/fixtures/mobile_apps.yml

```diff
@@ -719,3 +703,51 @@
     model: ""
   os_family: Mac
   browser_family: Unknown
+-
+  user_agent: 'Pic%20Collage/7.31.14 CFNetwork/889.9 Darwin/17.2.0'
+  os:
+    name: iOS
+    version: "11.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Pic Collage
+    version: 7.31.14
+  device:
+    type: ""
+    brand: Apple
+    model: ""
+  os_family: iOS
+  browser_family: Unknown
+-
+  user_agent: Pinterest for iOS/7.11 (iPhone7,2; 12.1.4)
+  os:
+    name: iOS
+    version: "7.11"
+    platform: ""
+  client:
+    type: mobile app
+    name: Pinterest
+    version: "7.11"
+  device:
+    type: smartphone
+    brand: Apple
+    model: iPhone 6
+  os_family: iOS
+  browser_family: Unknown
+-
+  user_agent: Slack/396839 CFNetwork/889.9 Darwin/17.2.0
+  os:
+    name: iOS
+    version: "11.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Slack
+    version: ""
```

Comment:
Slack version is not detected.

Reply:
this is done intentionally

Slack/396839 the versions don't match the appstore